app/views/users/_user.html.erb

```diff
@@ -0,0 +1,8 @@
+<div id="<%= dom_id user %>">
+  <p class="my-5">
+    <strong class="block font-medium mb-1">Name:</strong>
+    <%= user.name %>
+  </p>
+
+  <%#= link_to 'Edit this user', edit_user_path(user), class: "rounded-lg py-3 ml-2 px-5 bg-gray-100 inline-block font-medium" %>
```

Comment:
- [ ] It's a good convention to not have commented codes in the codebase as they make our codebase dirty and not professional. Kindly remove all commented codes from your codebase so your code looks clean and professional 🙃 😉

Reply:
I forgot about this one too. Thank you!